README.md

```diff
@@ -13,14 +13,14 @@ layout:
     visible: true
 ---
 
+# Kontrol
```

Comment:
I actually don't think we need it at all?

Reply:
okay

```suggestion
```